modules/ROOT/assets/overlays/metric.yaml

```diff
@@ -0,0 +1,94 @@
+overlay: 1.0.0
+
+info:
+  title: Overlay for Metrics REST API spec
+  version: 1.0.0
+
+extends: https://raw.githubusercontent.com/couchbase/sync_gateway/refs/heads/main/docs/api/metric.yaml
```

Comment:
As before, can this use:

 This gets post processed, can this use ../attachments/bundled-metric.yaml instead? This is also fixed to main which isn't necessarily going to be right. Right now main is 3.3 release.

Reply:
Updated as for `overlays/admin.yaml`